boards/arm/seeeduino_xiao/seeed_xiao_connector.dtsi

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright (c) 2021 Peter Johanson
+ *
+ * SPDX-License-Identifier: MIT
```

Comment:
Since you have the copyright, it would be nice to get this as Apache 2.0, since that's what Zephyr is licensed under generally speaking.

Reply:
Done.